### DISCUSSION TITLE
ANR error after upgrade to 15.1.1

### DISCUSSION THREAD

**Discussion by SERCHAT** [invertase/react-native-firebase#6417] [Q&A]
Hello everyone ,
I'm building an app for both devices Android/iOS and I'm using in my project below awesome react-native-firebase packages
app
auth
crashlitycs
dynamic links
messaging
google signin

I already have a published release app on google play store without any ANR error. After  upgrade my above packages to version 15.1.1 and publishing the app for internal release on google play console I started to see an ANR error what is say 
"ANR in com.google.android.googlequicksearchbox:interactor;PID: 2654;Broadcast of Intent { act=com.google.android.googlequicksearchbox.STOP_HOTWORD flg=0x10 pkg=com.google.android.googlequicksearchbox };".
I don't know if this ANR is related with the react-native-firebase packages, I doesn't change anything else in my project only I upgrade the above packages and republish on Google Play Store.

**Comment by mikehardy** [comment]
If the ANR is not in a stacktrace in this code or the underlying SDKs it should not have anything to do with this module
ANRs have a standard debugging path, involving use of strictmode in a developer environment to see what's slow during app startup: https://developer.android.com/reference/android/os/StrictMode

**Reply by SERCHAT** [reply]
Thank you @mikehardy have a good day! :)
